lldb/include/lldb/lldb-private-enumerations.h

```diff
@@ -172,10 +172,10 @@ typedef enum SwiftModuleLoadingMode {
 } SwiftModuleLoadingMode;
 
 // BEGIN SWIFT
-enum EnableSwiftCxxInterop {
-  eAutoDetectSwiftCxxInterop,
-  eEnableSwiftCxxInterop,
-  eDisableSwiftCxxInterop
+enum AutoBool {
```

Comment:
My C++ is lacking here – does this type autoconvert to bool? We want to make sure that `if ([](){ return eAutoBoolFalse;})` does the right thing and is a type check error.